package/CHANGELOG

```diff
@@ -23,6 +23,7 @@ Fixes
  * Fixes deprecation warning Array to scalar convertion. Replaced atan2() with np.arctan2() (Issue #4339)
  * Replaced mutable defaults with None and initialized them within
    the function to prevent shared state. (Issue #4655)
+ * Updated the docs for the self_distance_array function in package/MDAnalysis/lib/distances.py (Issue #4731) 
```

Comment:
I don't think this small documentation fix deserves an entry in the changelog. Can you please remove it?

Reply:
Yea, I was also not sure if I should add it. 
I removed it now